gamersnet_backend/routes/posts/getPosts.js

```diff
@@ -1,10 +1,44 @@
 'use strict';
 
 
-// include our function from the database to add post
-let {getAllPosts, getValidPosts} = require('../../persistence/posts');
+// include our function from the database to fetch posts
+let {getUserPosts, getAllPosts, getValidPosts} = require('../../persistence/posts');
 
-// this function handles the /post/getAllPosts/ endpoint
+let {verifyUserLoggedIn} = require('../utilities/tokenUtility')
+
+let {getUserIDFromToken} = require('../../persistence/tokens.js')
+
+// this function handles the /posts/listUserPosts/ endpoint
+async function listAllUserPosts(request, response) {
```

Comment:
i would suggest naming it listUserPosts(posts of a user)  instead of  listAllUserPosts(posts of all users)

Reply:
Alright!